docs/geedocs/5.2.4/answer/7160004.html

```diff
@@ -385,6 +403,16 @@ <h5>
           <td>Geserver fails to startup fully due to conflicting protobuf library</td>
           <td>Run <code>pip uninstall protobuf</code> to uninstall the protobuf library installed by pip. </td>
         </tr>
+        <tr>
+          <td>1012</td>
+          <td>Cutter produces incorrect very low resolution result from 2d mercator map</td>
+          <td>No current work around. </td>
```

Comment:
reword to "No known workaround"

Reply:
We are using in other known issues the wording "No current work around". If we change here we need to changes in other places to be consistent.

Reply:
Ok. I went through them and changed them all.